packages/mobile-native-mcp-server/src/workflow/magi/prd/nodes/prdFailure.ts

```diff
@@ -5,14 +5,14 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import { MCPToolInvocationData } from '../../../../common/metadata.js';
+import { MCPToolInvocationData } from '@salesforce/magen-mcp-workflow';
```

Comment:
Nit: it'd be nice to clean up these imports to all import from the same block, for `magen-mcp-workflow` imports. That's a repeated pattern across many of these modules.

Reply:
buh yeah we should probably have a lint rule for this. let me fix it and add one